.stylelintrc.json

```diff
@@ -64,8 +64,8 @@
     "selector-pseudo-class-parentheses-space-inside": "never",
     "selector-pseudo-element-case": "lower",
     "selector-pseudo-element-colon-notation": "single",
-    "selector-pseudo-element-no-unknown": true,
     "selector-type-case": "lower",
-    "selector-max-id": 0
+    "selector-max-id": 0,
```

Comment:
pls revert this file and I finished the review :)

Reply:
Done.